.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      matrix:
+        os:
+          - ubuntu-18.04
+          - ubuntu-20.04
+        python-version:
+          - '3.6'
+          - '3.7'
+          - '3.8'
+          - '3.9'
```

Comment:
Maybe add 3.10 and 3.11 while you're at it?

Reply:
I suppose, but I'd like to see this merged at all first - seeing as I've had PRs hanging for years in this repo I'm not quite having high hopes 😁